src/Twisto/Invoice.php

```diff
@@ -147,6 +151,40 @@ public function returnAll()
         $this->deserialize($data);
     }
 
+
+    /**
+     * Split invoice to new one
+     * @param ItemSplit[] $items
+     * @return Invoice
+     */
+    public function split()
```

Comment:
Tady chybí argument `$items`.

Reply:
No, úplně nechybí. Je to zvláštní. Nesedí to s dokumentací. V e-shop dokumentaci je metoda `splitItems`, která bere pole položek na novou fakturu.

Toto funguje, když se to použije takto:

```
$invoice = new Twisto\Invoice($twisto, $_POST['invoice_id']);
$invoice->items = array();
for($i = 0; $i < $_POST['items_split-TOTAL_FORMS']; $i++) {
    $item = new Twisto\ItemSplit($_POST['items_split-'. $i . '-product_id'], $_POST['items_split-'. $i . '-quantity']);
    array_push($invoice->items, $item);
}
$invoice->split();
```